app/services/get_system_metrics.rb

```diff
@@ -0,0 +1,96 @@
+# frozen_string_literal: true
+
+require 'sidekiq/api'
+
+class GetSystemMetrics
+  def initialize
+    @queues = YAML.load_file('config/sidekiq.yml')[:queues]
+                  .reject { |queue_name| queue_name == 'very_long_update' }
+    fetch_sidekiq_stats
+  end
+
+  def fetch_sidekiq_stats
+    stats = Sidekiq::Stats.new
+    {
+      enqueued_jobs: stats.enqueued,
+      active_jobs: stats.processes_size
+    }
+  end
+
+  def fetch_queue_management_metrics
+    queues = []
+    paused_queues = []
+    all_operational = true
+
+    @queues.each do |queue_name|
+      queue = Sidekiq::Queue.new(queue_name)
+      queues << get_queue_data(queue)
+
+      if queue.paused?
+        all_operational = false
+        paused_queues << queue_name
+      end
+    end
+
+    {
+      queues:,
+      paused_queues:,
+      all_operational:
+    }
+  end
+
+  def get_queue_data(queue)
+    {
+      name: queue.name,
+      size: queue.size,
+      status: get_queue_status(queue.name, queue.latency),
+      latency: convert_latency(queue.latency)
+    }
+  end
+
+  def get_queue_status(queue_name, latency)
+    latency_thresholds = {
```

Comment:
These should thresholds are constants, and should probably be implemented explicitly a constant object outside of the method.